docs/2-0-breaking-changes.md

```diff
@@ -34,6 +34,16 @@ This allows using human-in-the-loop nodes in the sub-workflow and use the result
 
 **Migration path:** Review any workflows that call sub-workflows and expect to receive the input to the sub-workflow. Update these workflows to handle the new behavior, where the parent-workflow receives the output from the end of the child-workflow instead.
 
+### Start node deprecated
```

Comment:
This should be removed, not deprecated. Deprecation and removal are different things. See the terminology [here](https://www.notion.so/n8n/Release-Process-2-0-Terminology-17a5b6e0c94f80ea9b83ede145a3713c)